api/examples/python/README.md

```diff
@@ -8,16 +8,13 @@ notifications.
 First, install the dependencies.
 
 ```
-sudo /usr/bin/python -m pip install websocket-client
-sudo /usr/bin/python -m pip install protobuf
-
+mkvirtualenv iterm2api
```

Comment:
IMHO modifying system python using sudo is more likely to lead to eventual frustration than using isolated environments, but I'll rewrite this section to be an alternative